tests/Unit/Domain/Creators/Test_Sphere.cpp

```diff
@@ -385,93 +372,68 @@ void test_sphere_boundaries_equiangular() {
+    CAPTURE(equiangular);
+    CAPTURE(radial_partitioning[array_index]);
+    CAPTURE(radial_distribution[array_index]);
+
+    const creators::Sphere sphere{inner_radius,
+                                  outer_radius,
+                                  sphericity,
+                                  initial_refinement,
+                                  initial_extents,
+                                  equiangular,
+                                  radial_partitioning[array_index],
+                                  radial_distribution[array_index]};
+
+    test_sphere_construction(sphere, inner_radius, outer_radius,
+                             radial_partitioning[array_index], false);
 
     const creators::Sphere sphere_boundary_condition{
         inner_radius,
         outer_radius,
         sphericity,
-        refinement,
-        grid_points_r_angular,
-        false,
+        initial_refinement,
+        initial_extents,
+        equiangular,
+        radial_partitioning[array_index],
+        radial_distribution[array_index],
         nullptr,
         create_boundary_condition()};
-    test_sphere_construction(
-        sphere_boundary_condition, inner_radius, outer_radius, sphericity,
-        false, grid_points_r_angular, {7, make_array<3>(refinement)}, true);
-  }
-}
 
-void test_sphere_factory_equidistant() {
-  INFO("Sphere factory equidistant");
-  for (bool use_boundary_conditions : {false, true}) {
-    const auto sphere = make_domain_creator(
-        "Sphere:\n"
-        "  InnerRadius: 1\n"
-        "  OuterRadius: 3\n"
-        "  InnerCubeSphericity: 0.1\n"
-        "  InitialRefinement: 2\n"
-        "  InitialGridPoints: [2,3]\n"
-        "  UseEquiangularMap: false\n"
-        "  TimeDependence: None\n",
-        use_boundary_conditions);
-    const double inner_radius = 1.0;
-    const double outer_radius = 3.0;
-    const double sphericity = 0.1;
-    const size_t refinement_level = 2;
-    const std::array<size_t, 2> grid_points_r_angular{{2, 3}};
-    test_sphere_construction(
-        dynamic_cast<const creators::Sphere&>(*sphere), inner_radius,
-        outer_radius, sphericity, false, grid_points_r_angular,
-        {7, make_array<3>(refinement_level)}, use_boundary_conditions);
+    test_sphere_construction(sphere_boundary_condition, inner_radius,
+                             outer_radius, radial_partitioning[array_index],
+                             true);
   }
 }
 
+// Check wedge neighbors have consistent directions & orientations, with time
+// dependence
 void test_sphere_factory_time_dependent() {
   INFO("Sphere factory time dependent");
-  // This name must match the hard coded one in UniformTranslation
-  const std::string f_of_t_name = "Translation";
-  const auto helper = [&f_of_t_name](
-                          const std::unordered_map<std::string, double>&
-                              expiration_times) {
+  const auto helper = [](const double expiration_time) {
```

Comment:
You don't need this lambda anymore. Just call the test function twice with two different times below.